src/main/java/org/icgc/argo/workflow_management/wes/model/NextflowMetadata.java

```diff
@@ -18,11 +18,11 @@
 
 package org.icgc.argo.workflow_management.wes.model;
 
+import java.util.Map;
 import lombok.Data;
-import nextflow.script.ScriptBinding.ParamsMap;
 
 @Data
 public class NextflowMetadata {
   private final NextflowWorkflowMetadata workflow;
-  private final ParamsMap params;
+  private final Map<String, Object> parameters;
```

Comment:
These are still not being indexed. In relay this is expected to be a [Map](https://github.com/icgc-argo/workflow-relay/blob/68ba12d675a175c8c4466e6d1fd6e2c3f30a45bd/src/main/java/org/icgc_argo/workflow/relay/model/nextflow/Metadata.java#L35)